Added RM & Potion Drops to "Current battle anticipation".

danielcrane opened the pull request.

danielcrane committed:
Added RM & Potion Drops to "Current battle anticipation".

cppisking commented:
Can you remove the 2 test files first?   `BrowserPac.js` and `Test`?

Fat-Chocobo commented:
Already removed Test I believe, I'll go ahead and remove BrowserPac.js now.
I had no idea if it was needed or not, so I just left it in. :)

danielcrane committed:
Removed test file.

Fat-Chocobo commented:
BrowserPac.js removed now.

danielcrane committed:
Added Inventory Filter, Inventory Export...

Added ability to filter inventory by item type (Weapon, Armour,
Accessory), added ability to export inventory to csv (by right-clicking
inventory area), added Gun weapon ID (so it doesn't show as "15" any
more), added ability to filter "Item Search" by "World" alone.

danielcrane committed:
Added Hairpin weapon type.

Added Hairpin weapon type to SchemaConstants.cs.

danielcrane committed:
Backend Tidy-up

danielcrane committed:
Current Dungeon & Current Battle Pages added.

Split "Current Dungeon" page into "Current Battle" and "Current Dungeon"
pages.

danielcrane committed:
Fixed some stuff.

danielcrane committed:
Backend Tidy-ups.

adelino-sdsu committed:
Added ItemDBUpdate.md

log of all Item updates to FFRKI database

adelino-sdsu committed:
Updated ItemDBUpdates.md

Added items from Tifa event

danielcrane committed:
Small Fix for uploading of drop info.

Added a small delay after the end of each battle before the item drops
are uploaded to the database, to avoid the request being throttled.

danielcrane committed:
Revert "Small Fix for uploading of drop info."

This reverts commit 849c2f0a986e1f3ecb001bc6a88dfe766d372e88.

danielcrane committed:
Added FFXIV and JSON Inventory Export.
